Form/ChoiceList/ModelChoiceList.php

```diff
@@ -20,7 +20,18 @@
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\PropertyAccess\PropertyPath;
 
+@trigger_error(
+    'The '.__CLASS__.' class is deprecated since 3.x'.
+    ' and will be removed in 4.0.'
```

Comment:
This part could be merged with the first line.

And normally, error message should be one-lined, but I guess I'm the only one of @sonata-project/contributors taking care of that. Meh.